devprofiler/src/reviewer.rs

```diff
@@ -96,15 +97,13 @@ fn process_blamelines(blamelines: &Vec<&str>, linenum: usize) -> HashMap<usize,
 		let mut author = wordvec[1];
 		let mut timestamp = wordvec[2];
 		let mut idx = 1;
-		// Check if the second value is an email address (enclosed in angle brackets)
-		if !author.starts_with('(') && !author.ends_with('>') {
-			// Shift the index to the next non-empty value
-			while idx < wordvec.len() && (wordvec[idx] == "" || !wordvec[idx].starts_with('(')){
-				idx += 1;
-			}
-			if idx < wordvec.len() {
-				author = wordvec[idx];
-			}
+		
+		// Check if the second value is an email address using the below regex
+		let pattern = r"[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}(?:\.[A-Za-z]{2,})*";
```

Comment:
- You probably need to escape `.`, `%`, `+` 
- And this also mandates that all the sections after the first `.` after the `@` symbol follow the `[A-Za-z]{2,}` pattern. Which might become a problem later.